.github/workflows/framework-tests.yaml

```diff
@@ -79,6 +81,8 @@ jobs:
         uses: actions/setup-go@v5
         with:
           go-version: "1.22"
+          # To suppress the "Restore cache failed" error, since there is no go.sum file here.
+          cache: false
```

Comment:
I guess there are 2 ways to go here:
1. no cache, always run pebble@master
2. `go get ...` and pin pebble

I think that @benhoyt intended the first option, so LGTM from me.
(Do speak up if that's not the case)